templates/vue/packagejsons/assemblyscript/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "near-blank-project",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "npm run build:contract && npm run build:web",
+    "build:contract": "asb",
+    "build:contract:debug": "asb --target debug",
+    "build:web": "node copy-dev-account.js && vue-cli-service build",
+    "dev:deploy:contract": "near dev-deploy",
+    "deploy:contract": "near deploy",
+    "deploy:pages": "gh-pages -d dist/",
+    "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
+    "prestart": "npm run build:contract:debug && npm run dev:deploy:contract",
+    "start": "echo The app is starting! It will automatically open in your browser when ready && npm run serve",
```

Comment:
This message is not true with the way this is currently written. Is there an option we can pass to `vue-cli-service serve` to automatically open the app?

Reply:
Looks like you can pass `vue-cli-service --open`, please do that in both package.json files

Reply:
Yes. Done.

Reply:
Done.